mipengine/node/monetdb_interface/monet_db_facade.py

```diff
@@ -135,7 +137,7 @@ def execute_queries_with_connection_handling(func, *args, **kwargs):
         finally:
             _MonetDBConnectionPool().release_connection(conn)
     else:
-        raise exc
+        raise timeout_exceptions
```

Comment:
wouldn't that try to `raise None` if the for loop finishes without raising any execptions?

Reply:
You are right. I just forgot to revert the changes on the monet_db_facade.
This will be addressed in a different branch.